backend_new/prisma/seed-helpers/listing-factory.ts

```diff
@@ -357,17 +359,19 @@ const unitFactory = (
           numBedrooms: i,
         },
       },
-      amiChart: {
-        create: {
-          items: {},
-          name: `listing: ${i} unit: ${j} amiChart: ${j}`,
-          jurisdictions: {
-            connect: {
-              id: jurisdictionId,
+      amiChart: amiChartId
+        ? { connect: { id: amiChartId } }
+        : {
+            create: {
+              items: {},
```

Comment:
suggest that `items: {}` be `items: []`, since it's a JSON array if populated.

Reply:
updated!